lib/liquid/standardfilters.rb

```diff
@@ -391,6 +391,17 @@ def round(input, n = 0)
       raise Liquid::FloatDomainError, e.message
     end
 
+    # Defaults are passed as nil so systems can easily override
+    def format_number(input, options = {})
+      options = {} unless options.is_a?(Hash)
+      precision = options['precision'] || 2
```

Comment:
Nit: even if there is no performance benefit, I would use constants for the default values. I find that generally makes the code easier to read.

Reply:
In this case, using constants will have a negative effect (albeit negligible).
Constants will be allocated at runtime even if the filter is never used in a template.